### PR TITLE
Minor edits to upgrading re trusted clusters and version

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/management/operations/upgrading.mdx [hunk 1]
```diff
@@ -49,6 +49,11 @@ Back up before upgrading. We have more instructions in [Backing up Teleport](./b
 
 ## Upgrade Sequence
 
+When upgrading multiple clusters:
```
Comment (zmb3) [inline docs/pages/management/operations/upgrading.mdx:52]: I'm not sure moving this first makes sense, as trusted clusters are a more complicated setup. My expectation would be that those who come to this doc are not familiar with the upgrade process, and therefore probably not yet at the point where they're running trusted clusters.

File: docs/pages/management/operations/upgrading.mdx [hunk 2]
```diff
@@ -83,22 +88,15 @@ precaution. This allows for a safe rollback in case the migration itself fails.
...
-upgrading resource services, you may upgrade in any sequence or at the same
+The Teleport Auth Service and Proxy Service are upgraded automatically within the specified
+window. When upgrading resource services, you may upgrade in any sequence or at the same
 time.
 
-When upgrading multiple clusters:
-
-1. First, upgrade the root cluster, i.e. the one that other clusters trust.
-2. Upgrade the Trusted Clusters.
+As documented in the Component compatibility the resource services should be the same major version or
```
Comment (zmb3) [inline docs/pages/management/operations/upgrading.mdx:98]: We refer to "the Component compatibility" as if someone should know what page that is. If they do already know what page we're talking about, they're probably familiar enough with things that they don't need this reminder.

Maybe something like this instead?

> Resource services should not be upgraded to a new major version
> until the Auth Service and Proxy Service have been upgraded.
> For more information on component compatibility, see <link>

I'm also not sure how I feel about the term "resource services." This is the first I've heard it.

Comment (ptgott) [reply docs/pages/management/operations/upgrading.mdx:98]: "Resource services" was one possible alternative we talked about when deciding on a general term for instances of the Kubernetes, SSH, etc. Service. The Core Concepts page uses "instance" or "agent" (https://goteleport.com/docs/core-concepts/#agent).

**Comment by ptgott** [comment]
@stevenGravy Just double-checking whether you plan to address Zac's feedback. I can take a look as well. Thanks!

**Comment by stevenGravy** [comment]
> @stevenGravy Just double-checking whether you plan to address Zac's feedback. I can take a look as well. Thanks!

Please take a look if you can!

**Comment by ptgott** [comment]
@stevenGravy Can you please either respond to Zac's feedback or close this PR? Thanks!

**Pull request closed without merging**